// @expo-google-fonts/albert-sans/index.js
// This file is generated by @expo-google-fonts/generator
// If you want to modify it, go to packages/generator/templates

export * from './useFonts';
export { default as __metadata__ } from './metadata.json';
export const AlbertSans_100Thin = require('./100Thin/AlbertSans_100Thin.ttf');
export const AlbertSans_200ExtraLight = require('./200ExtraLight/AlbertSans_200ExtraLight.ttf');
export const AlbertSans_300Light = require('./300Light/AlbertSans_300Light.ttf');
export const AlbertSans_400Regular = require('./400Regular/AlbertSans_400Regular.ttf');
export const AlbertSans_500Medium = require('./500Medium/AlbertSans_500Medium.ttf');
export const AlbertSans_600SemiBold = require('./600SemiBold/AlbertSans_600SemiBold.ttf');
export const AlbertSans_700Bold = require('./700Bold/AlbertSans_700Bold.ttf');
export const AlbertSans_800ExtraBold = require('./800ExtraBold/AlbertSans_800ExtraBold.ttf');
export const AlbertSans_900Black = require('./900Black/AlbertSans_900Black.ttf');
export const AlbertSans_100Thin_Italic = require('./100Thin_Italic/AlbertSans_100Thin_Italic.ttf');
export const AlbertSans_200ExtraLight_Italic = require('./200ExtraLight_Italic/AlbertSans_200ExtraLight_Italic.ttf');
export const AlbertSans_300Light_Italic = require('./300Light_Italic/AlbertSans_300Light_Italic.ttf');
export const AlbertSans_400Regular_Italic = require('./400Regular_Italic/AlbertSans_400Regular_Italic.ttf');
export const AlbertSans_500Medium_Italic = require('./500Medium_Italic/AlbertSans_500Medium_Italic.ttf');
export const AlbertSans_600SemiBold_Italic = require('./600SemiBold_Italic/AlbertSans_600SemiBold_Italic.ttf');
export const AlbertSans_700Bold_Italic = require('./700Bold_Italic/AlbertSans_700Bold_Italic.ttf');
export const AlbertSans_800ExtraBold_Italic = require('./800ExtraBold_Italic/AlbertSans_800ExtraBold_Italic.ttf');
export const AlbertSans_900Black_Italic = require('./900Black_Italic/AlbertSans_900Black_Italic.ttf');
